spec/index.bs

```diff
@@ -3765,17 +3765,18 @@ interface GPUQueue {
     void signal(GPUFence fence, GPUFenceValue signalValue);
 
     void writeBuffer(
-        ArrayBuffer sourceData,
-        GPUSize64 sourceOffset,
-        GPUBuffer destination,
-        GPUSize64 destinationOffset,
-        GPUSize64 size);
+        GPUBuffer buffer,
+        GPUSize64 bufferOffset,
+        ArrayBuffer data,
+        optional GPUSize64 dataOffset = 0,
+        optional GPUSize64 size = 0);
```

Comment:
I think I prefer the original order. These are very similar to copy commands and it's nice to keep all the copies in (source, destination) order.

Reply:
See the other comment.

Reply:
OK, no strong opinion.